org.adempiere.base/src/org/compiere/model/MMailText.java

```diff
@@ -505,4 +505,20 @@ public MUser getUser()
 		return m_user;
 	}
 
+	/**
+	 * 	@return Mail Header (Subject)
+	 */
+	public String getSuperMailHeader()
+	{
+		return super.getMailHeader();
+	}	//	getMailText
+
+	/**
+	 * 	@return Text used for Mail message
+	 */
+	public String getSuperMailText()
+	{
+		return super.getMailText();
+	}	//	getMailText
+

```

Comment:
this is not necessary, you can use `get_ValueAsString(MMailText.COLUMNNAME_MailHeader)` to get the value store in DB.